Add instructions for data collection and processing with Spectacular AI tools (#2668)
nerfstudio-project/nerfstudio: docs/quickstart/custom_dataset.md

ns-train nerfacto --data {output directory}
```

> (spectacularai=)

Reviewer: I think changing this to `(spectacularai)=` might fix the sphinx error "/home/runner/work/nerfstudio/nerfstudio/docs/quickstart/custom_dataset.md:22: WARNING: 'myst' reference target not found: spectacularai"

## Spectacular AI

Spectacular AI SDK and apps can be used to capture data from various devices:

 * iPhones (with LiDAR)
 * OAK-D cameras
 * RealSense D455/D435i
 * Azure Kinect DK

The SDK also records IMU data, which is fused with camera and (if available) LiDAR/ToF data when computing the camera poses. This approach, VISLAM, is more robust than purely image based methods (e.g., COLMAP) and can work better and faster for difficult data (monotonic environments, fast montions, narrow FoV, etc.).

Instructions:

1. Installation. With the Nerfstudio Conda environment active, first install the Spectacular AI Python library

```bash
pip install spectacularAI[full]
```

2. Install FFmpeg. Linux: `apt install ffmpeg` (or similar, if using another package manager). Windows: [see here](https://www.editframe.com/guides/how-to-install-and-start-using-ffmpeg-in-under-10-minutes). FFmpeg must be in your `PATH` so that `ffmpeg` works on the command line.

3. Data capture. See [here for specific instructions for each supported device](https://github.com/SpectacularAI/sdk-examples/tree/main/python/mapping#recording-data).
  

> 4. Process and export. Once you have recorded a dataset in Spectacular AI format an have it in `{data directory}` it can be converted into Nerfstudio format as:

Reviewer: ```suggestion
4. Process and export. Once you have recorded a dataset in Spectacular AI format and have it stored in `{data directory}` it can be converted into a Nerfstudio supported format with:
```
